go.mod

```diff
@@ -23,6 +23,8 @@ require (
 	github.com/metalmatze/signal v0.0.0-20210307161603-1c9aa721a97a
 	github.com/mitchellh/mapstructure v1.4.1
 	github.com/oklog/run v1.1.0
+	github.com/onsi/ginkgo v1.16.3 // indirect
+	github.com/onsi/gomega v1.13.0 // indirect
```

Comment:
Where is this coming from? :thinking:

Reply:
This is indirect dependency from a kube package used by the Openshift authentication.